Client/Telemetry/TelemetryWrapper.swift

```diff
@@ -273,7 +273,8 @@ class TelemetryWrapper {
         let currentWallpaper = LegacyWallpaperManager().currentWallpaper
 
         if case .themed = currentWallpaper.type {
-            GleanMetrics.WallpaperAnalytics.themedWallpaper[currentWallpaper.name].add()
+            // Need to lowercase the name for labeled counter
```

Comment:
Suggest adding a link to the "why" the lowercasing is needed.

```suggestion
            // Need to lowercase the name for labeled counter. Ref:
            // https://mozilla.github.io/glean/book/reference/metrics/index.html#label-format)
```